### PR TITLE
Add Server RedeDust

### PR DESCRIPTION
**Pull request by ViniciuszXL**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/dust/background.png
Comment (maaaathis) [inline minecraft_servers/dust/background.png:1]: Compress this file

File: minecraft_servers/dust/background@2x.png
Comment (maaaathis) [inline minecraft_servers/dust/background@2x.png:1]: Compress this file

File: minecraft_servers/dust/manifest.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+    "server_name": "dust",
+    "nice_name": "Dust",
+    "direct_ip": "rededust.com",
+    "server_wildcards": [
+        "%.rededust.com",
+        "%.mc-dust.com.br",
```
Comment (maaaathis) [inline minecraft_servers/dust/manifest.json:7]: Doesn't look like a real wildcard:
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/dust/manifest.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "server_name": "dust",
+    "nice_name": "Dust",
+    "direct_ip": "rededust.com",
+    "supported_languages": [
+        "pt-BR",
+        "pt-PT",
+        "en-US"
+    ],
+    "social": {
+        "web": "https://loja.rededust.com",
```
Comment (DoJapHD) [inline minecraft_servers/dust/manifest.json:11]: Use `web_shop` instead of `web`

File: minecraft_servers/dust/manifest.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        "youtube": "https://www.youtube.com/@rede_dust",
+        "discord": "https://discord.gg/rededust",
+        "tiktok": "rede_dust",
+        "twitter": "rede_dust",
+        "instagram": "rede_dust",
+        "twitch": "https://twitch.tv/rede_dust"
```
Comment (DoJapHD) [inline minecraft_servers/dust/manifest.json:17]: There's no twitch key, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**